src/main/java/org/minions/devfund/sergio/movie_database/Movie.java

```diff
@@ -0,0 +1,74 @@
+package org.minions.devfund.sergio.movie_database;
+
+import java.util.ArrayList;
+
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    public Movie(String name) {
+        this.name = name;
+        this.actors = new ArrayList<>();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Movie movie = (Movie) o;
+
+        if (Double.compare(movie.rating, rating) != 0) return false;
+        return name.equals(movie.name);
+    }
+
+    @Override
+    public int hashCode() {
+        int result;
+        long temp;
+        result = name.hashCode();
+        temp = Double.doubleToLongBits(rating);
+        result = 31 * result + (int) (temp ^ (temp >>> 32));
+        return result;
+    }
+
+    public int compareTo(Movie that) {
+        //write code here for compare name
+        return  this.getName().compareTo(that.getName());
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public ArrayList<Actor> getActors() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)